qemu/tests/x86_cpu_model.py

```diff
@@ -55,6 +46,56 @@ def run(test, params, env):
         guest_model = model_pattern % ""
     else:
         test.cancel("This host doesn't support cpu model %s" % model)
+    return cpu_model, guest_model, flags
+
+
+def check_flags(params, flags, test, session=None):
+    """
+    Check cpu flags
```

Comment:
It's more like,  check if flags are present on host or guest.

Reply:
thanks and update.